WPFHospitalEditor/Service/EquipmentServerService.cs

```diff
@@ -23,5 +24,13 @@ public IEnumerable<EquipmentDto> GetEquipmentByType(string equipmentType)
             return response.Data;
         }
 
+        public bool RelocateEquipment(EquipmentRelocationDto eqRealDto)
+        {
+            var client = new RestClient(AllConstants.ConnectionUrl);
+            var request = new RestRequest("Equipment/", Method.POST);
+            request.AddJsonBody(JsonConvert.SerializeObject(eqRealDto));
+            var response = client.Post<bool>(request);
+            return response.Data;
+        }
```

Comment:
You need rest endpoint in method RelocateEquipment in hospital microservice right? Rest path should be /api/hospital/equipment.

Reply:
Yes , you are right, I missed to change it after big refactor we recently had.